exo-setup/features/setup.feature

```diff
@@ -11,7 +11,8 @@ Feature: Setup of Exosphere applications
   Scenario: set up the "test" application
     Given a freshly checked out "test" application
     When running "exo-setup" in this application's directory
-    Then it has created the folders:
+    Then it finishes with exit code 0
```

Comment:
Please remove this line everywhere from this PR. Exit code 0 is checked by the previous line `When running ...`.